python/paddle/fluid/tests/unittests/test_dist_mnist_dgc_nccl.py

```diff
@@ -33,7 +33,10 @@ def test_dist_train(self):
         if fluid.core.is_compiled_with_cuda():
             self.check_with_place(
                 "dist_mnist.py",
-                delta=1e-5,
+                # FIXME(wangxi): DGC may need a new unit test because its algo flow 
```

Comment:
How to assert when DGC be disabled?

Reply:
Done. Use log files to judge whether DGC is disabled.